dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -297,6 +297,9 @@ public Workflow initializeWorkflow(String repositoryId, Workflow workflow) {
             workflow.setGitUrl(repository.getSshUrl());
             workflow.setLastUpdated(new Date());
             // Why is the path not set here?
+        } catch (GHFileNotFoundException e) {
+            LOG.info(gitUsername + ": GitHub reports file not found: " + e.getCause().getLocalizedMessage());
```

Comment:
I feel it's always good log the exception, just because you never know when it might come in handy.